src/olympia/lib/settings_base.py

```diff
@@ -20,6 +20,10 @@
     '.mozilla.net',
 ]
 
+PASSWORD_HASHERS = (
+    'django.contrib.auth.hashers.MD5PasswordHasher',
+)
```

Comment:
I personally would like to remove this alltogether and rely on django defaults. Move this overwrite to our `settings_test` for performance reasons but let's keep things safe for various other things that may rely on it?